internal/services/cognitive/cognitive_deployment_resource_test.go

```diff
@@ -131,6 +131,13 @@ func TestAccCognitiveDeployment_update(t *testing.T) {
 			),
 		},
 		data.ImportStep(),
+		{
+			Check: acceptance.ComposeTestCheckFunc(
```

Comment:
We need to specify the test config here
```suggestion
		{
			Config: r.dynamicThrottling(data),
			Check: acceptance.ComposeTestCheckFunc(
```